verify/trust/trust_test.go

```diff
@@ -0,0 +1,93 @@
+package trust
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+	"time"
+)
+
+// FakeHTTPSGetter is a test double for HTTPSGetter. It can hold a slice
+// of body and error responses so Get can be called multiple times and return
+// different values, as to simulate different scenarios.
+type FakeHTTPSGetter struct {
```

Comment:
Thanks! I didn't notice the mocks in `testing` folder.
I have moved the my mock over to `testing`. I opted to keep it as two seperate mocks, since combining it with the available `testing.Getter` to be both configurable in inputs and (mutliple) outputs seems like a lot of complexity for a mock. Let me know if you agree!
I also had to create `trust_test` package to break an import cycle back to `verify/trust` from `fakekds.go`.